signing/mcl/singlesig/testData/SingleSignTestVectorsSign.json

```diff
@@ -0,0 +1,36 @@
+{
+  "testVectors": [
```

Comment:
I think we would need to add to the testdata, for all vectors the following information
generatorPointG1 as a hex string, for G1
generatorPointG2 as a hex string, for G2
the default used dst as a hex string

same for the verify